build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
+group 'Sonshal'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+}
+
+sourceSets {
+    //useJUnitPlatform()
```

Comment:
Well, I don't see any changes here, please, read [my previous comment](https://github.com/DenisVerkhoturov/metamer/pull/1/files#r265299375) again=)
And then remove the `sourceSets` section and add the `test` section instead.
```groovy
test {
    useJUnitPlatform()
}
```

Tests still not working just because of this. Once you will make all the described changes you will be able to verify that did everything well running the following command:
```bash
gradle clean test
```
As a result you must see `BUILD SUCCESSFUL` and a new `index.html` file under the `build/reports/tests/test` directory. Open this file in a browser and you will see the following page:
![image](https://user-images.githubusercontent.com/11722244/54388361-9d5ecb80-46ae-11e9-80d9-ca0295978266.png)

Reply:
I did it, but I still can't run tests from the terminal :(
![1](https://user-images.githubusercontent.com/44375091/54489739-6d871200-48c0-11e9-982a-4e0c10810411.png)

But my build.gradle file is the same as Ira's

Reply:
Everything looks fine now, I was able to run `gradle clean test` locally in your project and faced no issues=) Do you still have this problem?